### DISCUSSION TITLE
PaddleOCR/main/ppstructure/model_train/train_layout

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14097] [Q&A]
# PaddleOCR/main/ppstructure/model_train/train_layout

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/main/ppstructure/model_train/train_layout.html?h=lcnet

**Comment by 9963KK** [comment]
我想问一下，如果我想要输出每个区域的四元组坐标，怎么调用呢？就比如说image标签框住区域的四元组坐标。
